contribute-to-docs/docs-release-process.md

```diff
@@ -53,6 +53,16 @@ to include the new version, and remove the oldest. Order matters, most recent fi
 
 1. PR the result to main.
 
+## Update the dot-release Job
+
+Edit the `.github/workflows/knative-dot-release-build.yaml` file to the just-cut release. For example, if v1.2 was the version just cut:
```

Comment:
```suggestion
Edit the `.github/workflows/knative-dot-release-build.yaml` file to the new release. For example, if you have just cut v1.2:
```

Reply:
I'm also wondering if it would be useful to make `.github/workflows/knative-dot-release-build.yaml` link to the file in GitHub.

Reply:
I'm also wondering if it would be useful to have `.github/workflows/knative-dot-release-build.yaml` link to the file in GitHub.

Reply:
:facepalm: yes, that would be good :smile: